src/contracts/Notification/UserPasswordReset.php

```diff
@@ -17,9 +17,7 @@
 use Symfony\Component\Notifier\Recipient\EmailRecipientInterface;
 use Symfony\Component\Notifier\Recipient\SmsRecipientInterface;
 
-final class UserPasswordReset
-extends Notification
-implements EmailNotificationInterface, SmsNotificationInterface, UserAwareNotificationInterface
+final class UserPasswordReset extends Notification implements EmailNotificationInterface, SmsNotificationInterface, UserAwareNotificationInterface
```

Comment:
Unless this comes from php-cs-fixer, I prefer the previous format better.

Reply:
this comes from cs-fixer, I would not fix that by hand by any means.